samples/nextjs/.env

```diff
@@ -36,11 +44,21 @@ SITECORE_API_HOST=
 # the resolved Sitecore API hostname + the `graphQLEndpointPath` defined in your `package.json`.
 GRAPH_QL_ENDPOINT=
 
+# Your Sitecore personalization decisions API endpoint.
+# The `PERSONALIZATION_DECISIONS_ENDPOINT` is required for personalizing using Sitecore.
+# By default, the endpoint is `personalizationDecisionsEndpointPath` defined in your `package.json`
+# which means it is proxied through Node instance to `sitecoreApiHost`.
+PERSONALIZATION_DECISIONS_ENDPOINT=
+
+# Determines whether tracking is enabled or not.
+# Default value: true
```

Comment:
It's not obvious where is this default value setup, can you describe it, or provide link to the place where it happens, for example? Is it in the sample app?

Reply:
The default value can be found in `samples/nextjs/scripts/generate-config.ts`

Reply:
@maximvsidorenko So, can you add line and specify that by default it's defined in 'src/temp/config.js'. In `generate-config.ts` you generate that config.